events/planner.py

```diff
@@ -0,0 +1,43 @@
+from events.models import OptionalMeetingDates, PossibleParticipant, Event
+
+
+class EventPlanner():
+
+    def run(self, event):
+        """ run the planner in an event.
+            Find the best possible date for the meeting,
+            then excute the choice and change the DB rows accordingly.
+        """
+        meetings_possibilities = self.find_meeting(event)
+
+        if len(meetings_possibilities) >= 1:
+            self.execute_choice(meetings_possibilities[0])
+
+    def find_meeting(self, event_id):
+        """ Find a meeting date with all the participants if possible.
+            The planner will look over all the possible dates and
+            will check if there is a possible date that works for everyone
+        """
+        results = list()
+        all_participants = PossibleParticipant.objects.get_all_possible_participants_amount(event_id)
```

Comment:
For consistency I think it is better to name the ..._amount function as ...._count

Reply:
Done 😄